pkg/logs/internal/tailers/file/tailer_test.go

```diff
@@ -45,7 +46,7 @@ func (suite *TailerTestSuite) SetupTest() {
 	var err error
 	suite.testDir = suite.T().TempDir()
 
-	suite.testPath = fmt.Sprintf("%s/tailer.log", suite.testDir)
+	suite.testPath = path.Join(suite.testDir, "tailer.log")
```

Comment:
I think you want `filepath` here, not `path` (and in pretty much any case)

Reply:
Yeah, I always make that mistake. Will update it today 👍🏽